File: lib/views/unique_visitors.rb
<% content_for :head do %>
  <script type="text/javascript" src="/javascripts/time-series.js"></script>
  <script type="text/javascript">
    $(function() {
      var graph = GOVUK.Insights.sixMonthTimeSeries("/unique-visitors.json", "#unique-visitors", {
        "series": {
          "govuk": {
            "lineClass": "main-line",
            "legendClass": "",
            "legend": "GOV.UK"
          },
          "directgov": {
            "lineClass": "dashed-line brown",
            "legendClass": "brown-text",
            "legend": "Directgov"
          },
          "businesslink": {
            "lineClass": "dashed-line purple",
            "legendClass": "purple-text",
            "legend": "Business Link"
          }
        }
      });
      graph.render();
    });
  </script>
<% end %>

<div class="module">
  <div class="module-header">
    <div class="info"></div>
    <div>
      <div class="small-text">Reach</div>
      <div>Weekly Unique Visitors for the last half year</div>
    </div>
  </div>
  <div style="clear: both"></div>
  <div id="unique-visitors"></div>
  <div class="small-text" style="padding: 18px 0">
    <span class="blue-text">Sources: Google Analytics, Celebrus, Omniture</span>
  </div>
</div>

File: lib/public/javascripts/modules/unique-visitors.js
$(function () {
    function showError() {
        $("#unique-visitors").append(GOVUK.Insights.Helpers.error_div);
    }

    $.ajax({
        url:"/unique-visitors.json",
        dataType:"json",
        success:function (data) {
            if (data == null) {
                showError();
            } else if (GOVUK.isSvgSupported()) {
                var graph = GOVUK.Insights.sixMonthTimeSeries("#unique-visitors", {
                    "series":{
                        "govuk":{
                            "lineClass":"main-line",
                            "gradient":true,
                            "legendClass":"",
                            "legend":"GOV.UK"
                        },
                        "directgov":{
                            "lineClass":"dashed-line brown",
                            "legendClass":"brown-text",
                            "legend":"Directgov"
                        },
                        "businesslink":{
                            "lineClass":"dashed-line purple",
                            "legendClass":"purple-text",
                            "legend":"Business Link"
                        }
                    },
                    "width":$("#unique-visitors").width()
                });
                graph.render(data);
            } else {
                $("#unique-visitors").append("<img src='/unique-visitors.png' style='max-width: none'></img>")
            }
        },
        error:showError
    });
});